ssg/build_cpe.py

```diff
@@ -82,11 +84,16 @@ def add_cpe_item(self, cpe_item):
     def _is_name(self, ref):
         return ref.startswith("cpe:")
 
+    def _is_parametrized(self, ref):
```

Comment:
This is a bad place for this method. It is a private method, we don't know that could possibly "ref" be, and we don't use `self` inside that method.
Most importantly, the class that knows about the syntax of parametrized platform units is the class that parses those definition strings, so it should be a static method of Symbol.